policy/modules/system/systemd.if

```diff
@@ -58,28 +60,612 @@ template(`systemd_role_template',`
 	# This domain is per-role because of the below transitions.
 	# See the systemd --user section of systemd.te for the
 	# remainder of the rules.
-	allow $1_systemd_t $3:process { setsched rlimitinh };
+	allow $1_systemd_t $3:process { setsched rlimitinh signal_perms };
+	allow $1_systemd_t self:netlink_kobject_uevent_socket create_socket_perms;
 	corecmd_shell_domtrans($1_systemd_t, $3)
 	corecmd_bin_domtrans($1_systemd_t, $3)
-	allow $1_systemd_t self:process signal;
+	allow $1_systemd_t self:process { getsched signal };
+
+	dontaudit $3 $1_systemd_t:unix_stream_socket rw_stream_socket_perms;
```

Comment:
this should eventually probably be allowed one way or another. the question is: how to implement this consistently and efficiently

i think this access is needed so that journald can catch stdout/stderr from "daemons". in other words without this journald might not catch the output of a command ran by systemd --user.

Reply:
Hmm, you're right. Without this access I cannot see the output of anything run with `systemctl --user` or `systemd-run --user`. I think that ultimately this needs a new interface to allow the proper accesses for `systemd --user` instances to run other executables and transition to those domains, as well as this access here to capture stdout/stderr. Though, that interface will need to be called for every application that we want `systemd --user` to be allowed to run.

Reply:
I think if we go that route we can explicitly allow this access for capturing from daemons running as the user `staff_t` (no transitions). There should probably be another separate patch series for adding this access for other applications that we want `systemd --user` to run. Something like `systemd_user_exec(staff, myapp_exec_t)` (no transition) and `systemd_user_run(staff, myapp_exec_t, myapp_t)` (with a transition).